peeringdb_server/middleware.py

```diff
@@ -46,28 +45,20 @@ def process_response(self, request, response):
 
         elif not request.COOKIES.get(settings.SESSION_COOKIE_NAME):
 
-            # request specifies no session, check if the request.path falls into the
-            # set of valid paths for new session creation
+            # allow session creation for all request paths except the API
 
-            NEW_SESSION_VALID_PATHS = [
-                reverse("login"),
-                reverse("register"),
-                reverse("username-retrieve"),
-                reverse("username-retrieve-initiate"),
-                reverse("reset-password"),
+            no_session_prefixes = [
+                "api/",
+                "api-auth/",
```

Comment:
Allowing everything except these has performance implications @peeringdb/oc will have to look at